src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -216,14 +213,14 @@ public StreamReader(String path, Encoding encoding, bool detectEncodingFromByteO
 
             _checkPreamble = (_preamble.Length > 0);
             _isBlocked = false;
-            _closable = !leaveOpen;
+            _leaveOpen = !leaveOpen;
```

Comment:
"closable" has the opposite meaning of "leaveOpen".  If you're going to change the name, you need to change conditions like this, too.  The leaveOpen functionality in general now appears to be broken.